spec/imsc-hrm.html

```diff
@@ -557,12 +566,16 @@ <h2>Paint Text</h2>
       <p class='note'>In the case where a property is <em>prohibited</em> in a profile of [[IMSC]], the computed value of the property
       specified in [[ttml2]] can be used.</p>
 
-      <p class='note'>While one-to-one mapping between characters and typographical glyphs is generally the rule in some scripts,
-      e.g. latin script, it is the exception in others. For instance, in arabic script, a character can yield multiple glyphs
-      depending on its position in a word. The Hypothetical Render Model always assumes a one-to-one mapping, but reduces the
-      performance of the glyph buffer for scripts where one-to-one mapping is not the general rule (see GCpy below).</p>
+      <p class='note'>While one-to-one mapping between <a>characters</a> and glyphs (as defined in [[i18n-glossary]])
```

Comment:
This reads as if the "one-to-one mapping" is defined in i18n-glossary. I think you mean that `glyph` is. Perhaps change `as defined in` to `using the definition of glyph from`. 

A better change might be to turn this paragraph over, e.g.:

>Note: The Hypothetical Render Model assumes that there is a one-to-one mapping between characters and glyphs. While this is common in some scripts (such as the Latin script), the actual relationship is more complex. Some scripts, such as Arabic, use different glyphs for a given character, depending on its position in a word. Some scripts require combining marks or use a sequence of code points to form a glyph. Cases exist where a given sequence of code points can have different glyph representations depending on context. This complexity is accounted for by reducing the performance of the glyph buffer for scripts where a one-to-one mapping is not the general rule (see GCpy below.



Nit: Insert `a` to make `While *a* one-to-one-mapping`

Reply:
Hopefully addressed in 1cba5f8 though right now GitHub isn't showing that on the PR view, so maybe there's a problem with the system. I made some changes in the first two sentences, with the goal of trying to avoid potential confusion, to end up with:

> Note
>
>The Hypothetical Render Model defines a one-to-one mapping between [characters](http://localhost:52332/spec/imsc-hrm.html#dfn-character) and [glyphs](http://localhost:52332/spec/imsc-hrm.html#dfn-glyph) (using the definition of glyph from this document). While a one-to-one mapping between [code points](http://localhost:52332/spec/imsc-hrm.html#dfn-code-point) and glyphs (using the definition of glyph from [[i18n-glossary](http://localhost:52332/spec/imsc-hrm.html#bib-i18n-glossary)]) is common in some scripts (such as the Latin script), the actual relationship is more complex. Some scripts, such as Arabic, use different glyphs for a given character, depending on its position in a word. Some scripts require combining marks or use a sequence of code points to form a glyph. Cases exist where a given sequence of code points can have different glyph representations depending on context. This complexity is accounted for by reducing the performance of the glyph buffer for scripts where a one-to-one mapping is not the general rule (see GCpy below).